Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3011,6 +3011,41 @@ internal static bool ShouldDrawWarningIconForBinding(string bindingPath)
 
 #if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
 
+#if UNITY_EDITOR
+        [InitializeOnLoad]
+        public static class PlayStateNotifier
```

Comment:
Its likely fine to keep this isolated but I also noticed we have both monitoring and evaluations of playmode state changes in https://github.com/Unity-Technologies/InputSystem/blob/develop/Packages/com.unity.inputsystem/InputSystem/NativeInputRuntime.cs#L341-L362 and https://github.com/Unity-Technologies/InputSystem/blob/develop/Packages/com.unity.inputsystem/InputSystem/InputManager.cs#L323

Wanted to share just for awareness. Might be better to keep this isolated.

Reply:
I see that other option now, thanks - was just focussed on InputSystem in isolation.

Reply:
I think we should keep it isolated, will resolve this